README.md

```diff
@@ -112,6 +113,7 @@ classDiagram
         number_trains : VectorData[int]
         intertrain_interval_in_ms : VectorData[float]
         power_in_mW : VectorData[float]
+        optical_fiber_locations_table_region : DynamicTableRegion
```

Comment:
```suggestion
        optical_fiber_locations_table_region : DynamicTableRegion[OpticalFiberLocationsTable]
```

This is not explicit in the spec, but it would be nice to be clear about this here.